### PR TITLE
Add Reference Guide

### PR DESCRIPTION
**Pull request by laksherthanlife**
Lets add a reference guide to make it clearer what each field is

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -73,6 +73,84 @@ This User Guide provides an in-depth documentation, so you can easily use and in
...
+
+Students are unique by name but are currently case-insensitive. This means you cannot add two or more students with the same name if they have the same case for this current version.
+
+#### Tag
+
+A Tag in ClassMonitor serves as a means to categorise students.
+
+Each student can have multiple tags. These Tags are optional.
+
+Feel free to add tags as you see fit to organize your inventory. Examples of how you may use a tag can include:
```
Comment (evedaktyl) [inline docs/UserGuide.md]: /inventory

File: docs/UserGuide.md [hunk 2]
```diff
@@ -28,6 +28,8 @@ This User Guide provides an in-depth documentation, so you can easily use and in
 
 ## Features
 
+## Quick Start
```
Comment (kwuunnn) [inline docs/UserGuide.md]: Minor bug

File: docs/UserGuide.md [hunk 3]
```diff
@@ -99,6 +101,84 @@ ClassMonitor collects and tracks the following data, allowing you to tailor your
...
+A student in ClassMonitor represents a student in one of your classes.
+
+The following are the fields for each student:
+
+* Student's Name
+* Student's Major
+* Student's Phone Number
+* Student's Email
+* Student's Stars (a measure of their class participation)
+* Student's Bolts (a measure of the classes the student was late or absent for)
```
Comment (kwuunnn) [inline docs/UserGuide.md]: just say absent

File: docs/UserGuide.md [hunk 4]
```diff
@@ -148,7 +228,7 @@ For the best user experience,
 * `PHONE_NUMBER`, `MAJOR`, `EMAIL`, `TAG` should be limited to 50 characters  or fewer
 * `NAME` should be limited to 30 characters or fewer<br>
 
-to ensure that all fields are fully visible on the screen without being cut off or truncated.
+to ensure that all fields are fully visible on the screen without being cut off or truncated,
```
Comment (kwuunnn) [inline docs/UserGuide.md]: should be `.`

File: docs/UserGuide.md [hunk 5]
```diff
@@ -397,7 +478,7 @@ Shows a pop-up message like below explaining how to access the help page.
 You can also close the program by using the **x** circle at the top left of the application's GUI.
 ```
 
-## Command summary
+## [Command summary](#command-summary)
```
Comment (kwuunnn) [inline docs/UserGuide.md]: Dont need to add this

File: docs/UserGuide.md [hunk 6]
```diff
@@ -432,7 +513,7 @@ Furthermore, certain edits can cause the ClassMonitor to behave in unexpected wa
 
 _Details coming soon ..._
 
-## FAQ
+## [FAQ](#faq)
```
Comment (kwuunnn) [inline docs/UserGuide.md]: Same for this

File: docs/placeholders/INDEX.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
+<!-- markdownlint-disable-file first-line-h1 -->
+The INDEX of an item is the number to the left of the item name in the Item List Box.
```
Comment (kwuunnn) [inline docs/placeholders/INDEX.md]: student not item

File: docs/placeholders/INDEX.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+
+```info
+* INDEX is a whole number larger than 0
+* INDEX is dependent on the item list currently displayed in the Item List Box
```
Comment (kwuunnn) [inline docs/placeholders/INDEX.md]: student not item

**Review by kwuunnn — COMMENTED** [review]
Quite a few problems and bugs, probably due to the merge conflicts.

File: docs/UserGuide.md [hunk 9]
```diff
@@ -99,6 +99,84 @@ ClassMonitor collects and tracks the following data, allowing you to tailor your
...
+A student in ClassMonitor represents a student in one of your classes.
+
+The following are the fields for each student:
+
+* Student's Name
+* Student's Major
+* Student's Phone Number
+* Student's Email
+* Student's Stars (a measure of their class participation)
+* Student's Bolts (a measure of the classes the student was late for)
```
Comment (kwuunnn) [inline docs/UserGuide.md]: the number of times a student is absent for class

instead of late

**Review by kwuunnn — COMMENTED** [review]
minor bug

**Pull request merged**